src/config/index.js

```
@@ -93,11 +57,19 @@ class Config {
    * Save the config in memory to disk
    */
   save() {
+    // Delete unsaved scripts
+    fse.readdir(SCRIPTS_DIRECTORY_PATH).then(scripts => {
```

Should add a `catch` that fails silently